autogen.sh

```diff
@@ -58,6 +58,11 @@ while ! npm install; do
   find node_modules -mindepth 1 -not -path node_modules/README -delete
 done
 
+# HACK: https://github.com/cockpit-project/cockpit/issues/12864
+# There's something about the ES6 version of PF4's collapsible.js that Edge
+# cannot stomach, so force the ES5 version
+cp node_modules/@patternfly/react-table/dist/js/components/Table/utils/decorators/collapsible.js node_modules/@patternfly/react-table/dist/esm/components/Table/utils/decorators/collapsible.js
```

Comment:
Shall we open an issue about that to them?

Reply:
FTR: This doesn't happen on current Edge, only on our two-year old version on our windows-10 image. Also, this feels a bit like an error on our side: They *do* ship an ES5 plain .js in dist/js, but our build system uses dist/esm/ without running it through babel. I'll soon send a better fix for this.